Include/zipimport.h

```diff
@@ -0,0 +1,14 @@
+#ifndef Py_ZIPIMPORT_H
+#define Py_ZIPIMPORT_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+PyAPI_DATA(PyTypeObject) PyZipImporter_Type;
```

Comment:
This and the rest of the file shouldn't be part of the stable ABI.

Reply:
But then how would someone be able to subclass or subtype the zip importer in their C code?

Reply:
Relying only on the stable ABI is an opt-in thing when you build an extension, so leaving this out of it won't prevent it from being available overall, just in a certain instance.